ValveResourceFormat/Resource/ResourceTypes/Texture.cs

```diff
@@ -149,7 +170,7 @@ public SKBitmap GenerateBitmap()
                             }
                         }
 
-                        return DDSImage.UncompressDXT1(Reader, Width, Height);
+                        return DDSImage.UncompressDXT1(Reader, Width, Height, NonPowerOfTwoWidth, NonPowerOfTwoHeight);
```

Comment:
Does it not work if you directly replace width/height?

Reply:
No, it would return mangled bits because there is a lot of work done on them (strides colors etc).  
Someone more skilled could optimize this later on.

Reply:
Note that all the extra work done in memory is less costly than outputting bigger padded images to disk - there are 5000+ in panorama/images, so this is actually faster overall.

Reply:
Scratch that, I gave up too soon. Fixed now (properly?)